test/src/test/integration/daemon/replication.rs

```diff
@@ -499,6 +499,12 @@ fn replication_includes_user() -> Result<(), anyhow::Error> {
         .await?;
 
         state::track(&alice.peer, project.urn(), bob.peer_id).await?;
+
+        // FIXME: we have to wait for these events otherwise track and fetch race. A
+        // similar double call is required in the track_peer test.
+        assert_fetched(&mut alice.events).await?;
+        assert_fetched(&mut alice.events).await?;
```

Comment:
Why is it two events that are awaited here?

Reply:
The same thing happens here https://github.com/radicle-dev/radicle-link/blob/master/test/src/test/integration/daemon/replication.rs#L454-L457. Now "why", I'm not too sure :grimacing: